README.md

```diff
@@ -30,15 +30,13 @@ jobs:
     runs-on: ubuntu-16.04
     strategy:
       matrix:
-        ruby: [ '2.x', '3.x' ]
+        ruby: [ '2.5.x', '2.6.x' ]
     name: Ruby ${{ matrix.ruby }} sample
     steps:
       - uses: actions/checkout@master
-      - name: Setup ruby
-        uses: actions/setup-ruby@v1
+      - uses: actions/setup-ruby@v1
         with:
           ruby-version: ${{ matrix.ruby }}
-          architecture: 'x64'
```

Comment:
right now, I think we need the x64 since we the current setup actions position is VM cache only and I believe it only has 64 bit

Reply:
I look at the action code and nowhere does it reference `architecture`.
https://github.com/actions/setup-ruby/blob/master/action.yml does not mention it either.
So my conclusion is the argument seems unused.

I also would assume in 99% cases people would want the default architecture, not e.g. 32-bit on a 64-bit Ubuntu worker as that just wouldn't run well.